src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/server/MixinMinecraftServer_AutoSaveInterval.java

```diff
@@ -0,0 +1,19 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft.server;
+
+import net.minecraft.server.MinecraftServer;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+
+import com.mitchej123.hodgepodge.config.TweaksConfig;
+
+@Mixin(MinecraftServer.class)
+public class MixinMinecraftServer_AutoSaveInterval {
+
+    @ModifyConstant(method = "tick", constant = @Constant(intValue = 900))
+    private int modifyAutoSaveInterval(int constant) {
```

Comment:
Please prefix the method with `hodgepodge$`. It makes stacktraces more readable because you can't rely on the package name anymore to figure out where to look.